### ISSUE TITLE
Alarm doesn't generate vibration (Vollaphone X)

### ISSUE THREAD

**Issue by j2g2rp** [HelloVolla/ubuntu-touch-beta-tests#113]
Device: Vollaphone X
Software version: OTA 19
Vibration when Alarm is ringing isn't woriking.

**Comment by JamiKettunen** [comment]
Can confirm also on Volla Phone 22 (mimameid), and presumably this would also be an issue on Volla Phone (yggdrasil) but I didn't test yet.

The problem comes down to whatever component is trying to activate the alarm vibration is still trying to use `usensord` which the functionality of is (mostly) replaced by `hfd-service` in Halium 9+ ports; the following can be seen in `~/.cache/upstart/usensord.log` when the alarm goes off and alarm vibration is enabled in the Clock application settings:
```
uSensord: 2022/08/31 13:10:00 haptic.go:236: Error opening haptic device
uSensord: 2022/08/31 13:10:02 haptic.go:236: Error opening haptic device
uSensord: 2022/08/31 13:10:04 haptic.go:236: Error opening haptic device
uSensord: 2022/08/31 13:10:06 haptic.go:236: Error opening haptic device
uSensord: 2022/08/31 13:10:08 haptic.go:236: Error opening haptic device
```
<p><details>
<summary>And in "dbus-monitor" (click to show/hide log output)</summary>
<pre><code>method call time=1661946360.014356 sender=:1.60 -> destination=:1.45 serial=395 path=/org/freedesktop/Notifications; interface=org.freedesktop.Notifications; member=Notify
   string "indicator-datetime-service"
   uint32 0
   string "alarm-clock"
   string "Alarm Wed,  2:46 PM"
   string "Alarm"
   array [
      string "none"
      string "OK"
      string "snooze"
      string "Snooze"
   ]
   array [
      dict entry(
         string "x-canonical-non-shaped-icon"
         variant             string "true"
      )
      dict entry(
         string "x-canonical-snap-decisions"
         variant             string "true"
      )
      dict entry(
         string "x-canonical-snap-decisions-timeout"
         variant             int32 600000
      )
      dict entry(
         string "x-canonical-private-affirmative-tint"
         variant             string "true"
      )
   ]
   int32 -1
 ...
 method call time=1661946370.029953 sender=:1.60 -> destination=com.canonical.usensord serial=423 path=/com/canonical/usensord/haptic; interface=com.canonical.usensord.haptic; member=VibratePattern
   array [
      uint32 1000
      uint32 1000
   ]
   uint32 1
method call time=1661946370.031912 sender=:1.5 -> destination=org.freedesktop.DBus serial=54 path=/org/freedesktop/DBus; interface=org.freedesktop.DBus; member=GetConnectionCredentials
   string ":1.60"
method return time=1661946370.032315 sender=org.freedesktop.DBus -> destination=:1.5 serial=55 reply_serial=54
   array [
      dict entry(
         string "ProcessID"
         variant             uint32 5237
      )
      dict entry(
         string "UnixUserID"
         variant             uint32 32011
      )
      dict entry(
         string "LinuxSecurityLabel"
         variant             array of bytes "unconfined" + \0
      )
   ]
error time=1661946370.034892 sender=:1.3 -> destination=:1.60 error_name=com.canonical.usensord.Error reply_serial=423
   string "open /sys/class/timed_output/vibrator/enable: no such file or directory"</code></pre>
</details></p>

From the looks of things the upstream component at fault seems to be https://github.com/ubports/indicator-datetime (at least on 16.04).

**Comment by JamiKettunen** [comment]
https://github.com/AyatanaIndicators/ayatana-indicator-datetime/pull/101 fixes this on Ubuntu Touch 20.04, we can incorporate a similar codepath for the [legacy 16.04 `indicator-datetime`](https://github.com/ubports/indicator-datetime) to also try `hfd-service` instead of just `usensord`, hopefully we can get that in for the next-next 16.04 OTA-25 :)

**Comment by JamiKettunen** [comment]
This is fixed on 20.04 and 16.04 is EoL with no further changes getting in for almost a year already, closing this now.
